pyproject.toml

```diff
@@ -135,6 +135,41 @@ source = "https://github.com/scipy/scipy"
 download = "https://github.com/scipy/scipy/releases"
 tracker = "https://github.com/scipy/scipy/issues"
 
-
 [tool.doit]
 dodoFile = "dev.py"
+
+
+[tool.cibuildwheel]
+skip = "cp36-* cp37-* pp* *_ppc64le *_i686 *_s390x *-musllinux*"
+build-verbosity = "3"
+before-test = "pip install -r {project}/tools/wheels/test_requirements.txt"
+test-command = "bash {project}/tools/wheels/cibw_test_command.sh {project}"
+
+[tool.cibuildwheel.linux]
+manylinux-x86_64-image = "manylinux2014"
+manylinux-aarch64-image = "manylinux2014"
+before-build = "bash {project}/tools/wheels/cibw_before_build_linux.sh {project}"
+test-skip = "*_aarch64"
+
+[tool.cibuildwheel.macos]
+# If universal2 wheels are wanted use delocate-fuse
+# TODO figure out arm64 cross compilation
```

Comment:
This TODO can be removed, it seems to work now

Reply:
I think the `universal2` comment can go as well. We don't want those anymore.

Reply:
cibuildwheel isn't making the macosx_arm64 wheels, that's done with some hand-rolled code. When https://github.com/pypa/cibuildwheel/issues/1227 is addressed we can try again.